baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -198,9 +202,12 @@ def handle(self, message: confluent_kafka.Message) -> None:
                     self.on_success_fn(context, data, message)
 
                 if message_latency is not None:
-                    context.metrics.timer(f"{self.name}.{topic}.latency").send(message_latency)
+                    context.metrics.timer(f"{self.name}.{topic}.latency").send(
+                        message_latency
+                    )
 
-                context.metrics.gauge(f"{self.name}.{topic}.offset.{partition}").replace(offset)
+                context.metrics.gauge(f"{self.name}.{topic}.offset.{
+                                      partition}").replace(offset)
```

Comment:
Is this a syntax error? Or at least, Black can't parse it:

```
error: cannot format baseplate/frameworks/queue_consumer/kafka.py: Cannot parse: 209:39:
                 context.metrics.gauge(f"{self.name}.{topic}.offset.{
```

Reply:
Suprisingly other files failed when being formatted (even with black). So I just disabled my formatter in general.